cli/delete.go

```diff
@@ -79,6 +79,7 @@ func deleteRecordCommand(t *core.Timetrace) *cobra.Command {
 				return
 			}
 
+			showRecord(record)
 			if !confirmed {
 				if !askForConfirmation() {
 					out.Err("Record NOT deleted")
```

Comment:
```suggestion
					out.Info("Record NOT deleted.")
```

This is just a suggestion: If the user has been asked whether they want to delete the record, and they decide to abort the deletion, is it really an error? I'd probably use something like `out.Info` instead.

Everything else looks good to me! 👍

Reply:
Agree with that, it's not an error